scanamo/src/main/scala/org/scanamo/request/ScanamoRequest.scala

```diff
@@ -71,7 +71,7 @@ object ScanamoQueryOptions {
 case class RequestCondition(
   expression: String,
   attributeNames: Map[String, String],
-  dynamoValues: Option[DynamoObject]
+  dynamoValues: DynamoObject
```

Comment:
This is the change to the model that drives all the simplifications we see in the rest of the PR.